app/components/post/post.js

```diff
@@ -88,29 +90,19 @@ export default class Post extends PureComponent {
 
     goToUserProfile = () => {
         const {intl} = this.context;
-        const {navigator, post, theme} = this.props;
-        const options = {
-            screen: 'UserProfile',
-            title: intl.formatMessage({id: 'mobile.routes.user_profile', defaultMessage: 'Profile'}),
-            animated: true,
-            backButtonTitle: '',
-            passProps: {
-                userId: post.user_id,
-            },
-            navigatorStyle: {
-                navBarTextColor: theme.sidebarHeaderTextColor,
-                navBarBackgroundColor: theme.sidebarHeaderBg,
-                navBarButtonColor: theme.sidebarHeaderTextColor,
-                screenBackgroundColor: theme.centerChannelBg,
-            },
+        const {actions, post} = this.props;
+        const screen = 'UserProfile';
+        const title = intl.formatMessage({id: 'mobile.routes.user_profile', defaultMessage: 'Profile'});
+        const passProps = {
+            userId: post.user_id,
         };
 
         Keyboard.dismiss();
         requestAnimationFrame(() => {
             if (Platform.OS === 'ios') {
-                navigator.push(options);
+                actions.goToScreen(screen, title, passProps);
             } else {
-                navigator.showModal(options);
+                actions.showModal(screen, title, passProps);
```

Comment:
same here, lets check this is still needed for Android, if not just push the screen